src/filteredMparticleUser.js

```diff
@@ -67,10 +67,9 @@ export default function filteredMparticleUser(
             return userAttributesLists;
         },
         getAllUserAttributes: function() {
+            // TODO: May not need to make a copy since Store should return a copy
```

Comment:
Note te self: we need this empty object to create a new subset of attributes for filtering.